Code/Tools/ProjectManager/Source/GemCatalog/GemCatalogScreen.h

```diff
@@ -62,14 +64,22 @@ namespace O3DE::ProjectManager
 
     private slots:
         void HandleOpenGemRepo();
-
+        void UpdateAndShowGemCart(QWidget* cartWidget);
+        void ShowInspector();
 
     private:
+        enum RightPanelWidgetOrder
+        {
+            Inspector = 0,
+            Cart
+        };
+
```

Comment:
I'm not sure this enum is needed if you use QStackedWidget::setCurrentWidget()

Reply:
Planning to do a pass later to change ownership around and use some shared pointers for the cart and download controller because tight now they have tons of raw pointers saved all over the place.